docs/SupportedCommands/Commands.md

```diff
@@ -50,27 +50,33 @@
 
 | Proxied Command (Android only)    |
 |-----------------------------------|
-| `getNetworkConnection`<sup>2</sup>|
+| `getNetworkConnection`            |
 | `isAppInstalled`                  |
 | `isLocked`                        |
 | `longPressKeyCode`                |
 | `pressKeyCode`                    |
-| `setNetworkConnection`<sup>2</sup>|
+| `setNetworkConnection`            |
 | `toggleLocationServices`          |
 | `unlock`                          |
 
-
+### Attributes
 The following attributes can be queried using `attribute`
-|          Attribute           |          Engine Version Support          |
+
+Attributes can be appended to the strategy value with following format: `[@attributeType='attributeValue']`
```

Comment:
Can you also mention that its by (name, class, id) and no support for xpath + an example. Thoughts?